xlive/Blam/Cache/TagGroups/scenario_structure_bsp_definition.hpp

```diff
@@ -7,7 +7,7 @@
 * group_tag : sbsp
 * header size : 572
 * *********************************************************************/
-struct s_scenario_structure_bsp_group_definition :TagGroup<'sbsp'>
+struct s_scenario_structure_bsp_group_definition
```

Comment:
Why are we removing the taggroup typing that's used for grabbing and validating tags

Reply:
Caused compile issues, and there's a better way to grab the tag group type of a tag rather than embed it in the definition

Reply:
works for me as long as there is still a quick method of verification